### PR TITLE
DGDS-298/297 specify onclick actions rasterlayers

### PR DESCRIPTION
**Pull request by CindyvdVries**

### CODE REVIEW

File: components/v-mapbox-components/v-mapbox-info-text-layer.vue [hunk 1]
```diff
@@ -0,0 +1,61 @@
+<template>
+  <div />
```
Comment (SiggyF) [inline components/v-mapbox-components/v-mapbox-info-text-layer.vue]: use 
```
render()  {
  return null
}
```

File: components/v-mapbox-components/v-mapbox-info-text-layer.vue [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+</template>
+
+<script>
+export default {
+  name: 'VMapboxSelectedPointLayer',
+  props: {
+    geometry: {
```
Comment (SiggyF) [inline components/v-mapbox-components/v-mapbox-info-text-layer.vue]: why not required?

File: components/v-mapbox-components/v-mapbox-info-text-layer.vue [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+      },
+    },
+    message: {
+      type: String,
+    },
+  },
+  data() {
+    return {
+      map: null,
+      selectedLayer: {
```
Comment (SiggyF) [inline components/v-mapbox-components/v-mapbox-info-text-layer.vue]: comments?

File: layouts/default.vue [hunk 4]
```diff
@@ -21,14 +25,17 @@
           :active-theme="activeTheme"
           @select-locations="selectLocations"
         ></v-mapbox-vector-layer>
-        <v-mapbox-raster-layer :options="rasterLayer"> </v-mapbox-raster-layer>
+        <v-mapbox-raster-layer
+          :options="rasterLayer"
+          @click:raster="getFeatureInfo"
```
Comment (SiggyF) [inline layouts/default.vue]: why not just `@click`?

File: layouts/default.vue [hunk 5]
```diff
@@ -170,6 +204,41 @@ export default {
...
+      if (!this.getActiveRasterLayer) {
+        this.removeInfoText()
+        return
+      }
+      const parameters = {
+        imageId: this.activeRasterData.imageId,
+        bbox: bbox,
+        band: bands[this.getActiveRasterLayer],
+      }
+      fetch(`${process.env.HYDRO_ENGINE}/get_feature_info`, {
```
Comment (SiggyF) [inline layouts/default.vue]: This is too hidden, move it up in the code.

Comment (SiggyF) [reply layouts/default.vue]: Don't forget to update the `.env.example` and to update the deployment configs.

File: nuxt.config.js [hunk 6]
```diff
@@ -11,6 +11,7 @@ export default {
   env: {
     MAPBOX_ACCESS_TOKEN: process.env.MAPBOX_ACCESS_TOKEN,
     SERVER_URL: process.env.SERVER_URL,
+    HYDRO_ENGINE: process.env.HYDRO_ENGINE,
```
Comment (SiggyF) [inline nuxt.config.js]: `_URL`?

File: layouts/default.vue (hunk 5 above)
Comment (avgils) [reply layouts/default.vue]: The url should come from backend.

Comment (SiggyF) [reply layouts/default.vue]: fixed in travis and .env

**Pull request merged**